### PR TITLE
Flinkster Carsharing via MOQO + latest-parking cache

### PR DESCRIPTION
**Pull request by ThorstenFroehlinghaus**
The MOQO converter considers missing attribute values of the flinkster data: vehicle_type, license, fuel_level, street.

The latest-parking cache of all cars ensures, that cars which are in use are reported as 'is_reserved' at their station.

### CODE REVIEW

File: x2gbfs/providers/moqo.py [hunk 1]
```diff
@@ -39,6 +44,8 @@ class MoqoProvider(BaseProvider):
     DEFAULT_PRICING_PLAN_PATTERN = '{vehicle_type}_hour_daytime'
     MINIMUM_REQUIRED_AVAILABLE_TIMESPAN_IN_SECONDS = 60 * 60 * 3  # 3 hours
 
+    cars_latest_parking_cache: dict[str, int] = {}
```
Comment (hbruch) [inline x2gbfs/providers/moqo.py:47]: could you add a comment, why this cache is needed?
And is storing the ID as int intended, as for `stations_status` and `station_information` the id is converted to a string.

File: x2gbfs/providers/moqo.py [hunk 2]
```diff
@@ -182,8 +195,17 @@ def _extract_from_vehicles(
             },
         }
 
+        if vehicle.get('license'):
+            gbfs_vehicle['license_plate'] = re.split(r'[(|]', vehicle['license'])[0].strip()
+
         if vehicle.get('latest_parking') is not None and vehicle.get('latest_parking', {}).get('id') is not None:
```
Comment (hbruch) [inline x2gbfs/providers/moqo.py:201]: As the id is used twice, for the existence check and assignment, you could assign it to a variable,
`latest_parking = vehicle.get('latest_parking', {}).get('id')` will work in all three cases:  if the `latest_parking` does not exist, it's value is or set.

**Review by hbruch — CHANGES_REQUESTED** [review]
`station_id` should be string, not int

File: x2gbfs/providers/moqo.py (hunk 2 above)
Comment (ThorstenFroehlinghaus) [reply x2gbfs/providers/moqo.py]: This expression leads to an exception:

  File "/home/ipl/ipl46_flinkster/x2gbfs/providers/moqo.py", line 202, in _extract_from_vehicles
    latest_parking_id = vehicle.get('latest_parking', {}).get('id')
AttributeError: 'NoneType' object has no attribute 'get'

For this reason, I used two variables latest_parking and latest_parking_id

Comment (hbruch) [reply x2gbfs/providers/moqo.py]: Ok.

**Review by hbruch — APPROVED** [review]
LGTM

**Pull request merged**